src/cli/utils/onedrive.go

```diff
@@ -25,6 +26,20 @@ type OneDriveOpts struct {
 	Populated PopulatedFlags
 }
 
+// OneDriveOptions constructor for OneDriveOpts
+func OneDriveOptions(users, paths, fileNames []string, command *cobra.Command) OneDriveOpts {
+	return OneDriveOpts{
+		Users:              users,
+		Paths:              paths,
+		Names:              fileNames,
+		FileCreatedAfter:   FileCreatedAfter,
+		FileCreatedBefore:  FileCreatedBefore,
+		FileModifiedAfter:  FileModifiedAfter,
+		FileModifiedBefore: FileModifiedBefore,
+		Populated:          GetPopulatedFlags(command),
+	}
+}
```

Comment:
I'd prefer we keep with the original design (ie: direct struct creation, not a constructor).  This constructor utilizes shared global state that could make testing, even unit testing, more error prone.

Reply:
Resovled with [110b78a](https://github.com/alcionai/corso/pull/2679/commits/110b78af9ba3e0a55eaf8a1b45fd0814502c0f1f)

Reply:
Will pass in the strings Temporal modification rather than utilize shared state.

Reply:
Reverted to the original design. Current as of [b25b0d4](https://github.com/alcionai/corso/pull/2679/commits/b25b0d450180f48d5fcf79a262ffbdf355a73a94)